wp-content/themes/goonj-crm/assets/styles/goonj-main.scss

```
@@ -552,4 +605,37 @@
   .d-xl-flex {
     display: flex !important;
   }
+}
+
+// Contribution amount options (the price-set radios) on monetary contribution
+// pages — /contribute/website-user, /contribute/team-5000, the RAHAT pages, etc.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)